include/RAJA/exec-cuda/raja_cuda.hxx

```diff
@@ -144,14 +144,22 @@ struct cuda_exec_async : public cuda_exec<BLOCK_SIZE, true> {
 ///
 ///////////////////////////////////////////////////////////////////////
 ///
-template <size_t BLOCK_SIZE>
+template <size_t BLOCK_SIZE, bool Async = false>
```

Comment:
What do we gain by having the bool parameter here?  We explicitly define cuda_reduce_async, etc. policies below.